### PR TITLE
Nsng 593 gtm

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
Needs to be tested and consulted

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -29,11 +28,22 @@ export default defineNuxtConfig({
       apiUrl: process.env.NUXT_API_URL,
     },
   },
+  gtm: {
+    id: process.env.NUXT_APP_GTM_ID || '',
+    enabled: process.env.NUXT_APP_GTM_ID === 'true' ? true : false, // Enable GTM tracking
```
Comment (FrantisekMichalSebestyen) [inline nuxt.config.ts:33]: @igor-kamil could not come up with anything better

Comment (igor-kamil) [reply nuxt.config.ts:33]:
```suggestion
    enabled: process.env.NUXT_APP_GTM_ENABLED === 'true' ? true : false, // Enable GTM tracking
```
would be better 😂 

however `enabled` doesn't seems to really work anyway. I've found this [workaround](https://github.com/zadigetvoltaire/nuxt-gtm/issues/26#issuecomment-2190851752)

**Review by igor-kamil — COMMENTED** [review]
TBH, I am not sure, whether `@zadigetvoltaire/nuxt-gtm` is a good choice.
  apparently [not maintained anymore](https://github.com/zadigetvoltaire/nuxt-gtm/issues/43). Also it [breaks the app](https://github.com/zadigetvoltaire/nuxt-gtm/issues/24) (=returns 500) until you define the GTM ID (what will make it impossible even to run tests). 

Also it seems to be not such a big deal to implement it without any package -> https://scripts.nuxt.com/scripts/tracking/google-tag-manager 

But to sum it up - I consider it as your project and as an owner you can still merge this (please just accept the suggestion - right now you don't use `NUXT_APP_GTM_ENABLED` in the code at all).

Otherwise I suggest to implement it without requiring to set `GTM_ID`

**Comment by FrantisekMichalSebestyen** [comment]
Closed in favor of #24

**Pull request closed without merging**